cluster/manifests/skipper/deployment.yaml

```diff
@@ -1,7 +1,7 @@
 {{/* image-updater-bot detects *image variables so use print to disable it for main image */}}
 
 {{ $main_image := print "container-registry.zalando.net/teapot/skipper-internal:" "v0.21.211-1033" }}
-{{ $canary_image := "container-registry.zalando.net/teapot/skipper-internal:v0.21.212-1034" }}
```

Comment:
I think we should discuss how to update canary and main image. Do we want to have it a weekly schedule or something like that?
I wonder what can happen if we update canary 2x to X and Y from V and go straight from V to Y in main_image.

Reply:
We currently have one update in the pipeline https://github.com/zalando-incubator/kubernetes-on-aws/pull/8295